src/applications/personalization/dashboard/components/benefit-payments-v2/BenefitPaymentsV2.jsx

```diff
@@ -6,6 +6,8 @@ import PaymentsCardV2 from './PaymentsCardV2';
 import DashboardWidgetWrapper from '../DashboardWidgetWrapper';
 import IconCTALink from '../IconCTALink';
 import recordEvent from '~/platform/monitoring/record-event';
+import { canAccess } from '../../selectors';

```

Comment:
I don't know where this would be best located, but I was thinking maybe we should have a shared selectors area in `personalization/common` or somewhere similar. 

This `canAccess` will most likely get used in the profile as well as the dashboard. It's fine where it is at right now, but just a thought I had.